pkg/deploy/defaults.go

```diff
@@ -30,7 +30,6 @@ const (
 	DefaultPluginRegistryPullPolicy     = "Always"
 	DefaultPluginRegistryMemoryLimit    = "32Mi"
 	DefaultPluginRegistryMemoryRequest  = "16Mi"
-	DefaultCodereadyPluginRegistryUrl   = "https://che-plugin-registry.openshift.io"
```

Comment:
In fact I had left this line for the Che 6 CRW case, where plugin registry was already used. But since we now manage Che 6 from a distinct branch, I assume we're OK to remove this line here. However did you check this compiles correctly after removing this line ?

Reply:
nope, wanted to discuss how much to change first :)